plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/LabelImageLoadJob.java

```diff
@@ -51,7 +51,7 @@
   @Override
   protected IStatus run(IProgressMonitor monitor) {
     ImageDescriptor descriptor = ImageDescriptor.createFromURL(imageUrl);
-    image = descriptor.createImage();
+    image = descriptor.createImage(display);
```

Comment:
This is still creating the image on a non-SWT thread.  I thin this code can just use `ImageDescriptor#getImageData()`.